src/views/Machote.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+  <section class="machote">
+    <div class="machote__go-back-btn-wrapper">
+      <a class="machote__go-back-btn" @click="$router.go(-1)">Volver</a>
```

Comment:
Two problems with this:
- It doesn't work if user comes from external site (i.e. FB or Twitter).
- Cursor doesn't change to `pointer` because anchor doesn't have an `href`. Use CSS to force it or change how this element is implemented.